android/java/res/layout/brave_news_settings.xml

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) 2022 The Brave Authors. All rights reserved.
+     This Source Code Form is subject to the terms of the Mozilla Public
+     License, v. 2.0. If a copy of the MPL was not distributed with this file,
+     You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
correct please `http` to `https`. It's not only in that file, it's inside most xml files.